### PR TITLE
Group rework

### PR DESCRIPTION
**Pull request by Aqa-Ib**
Changegroupactive backwards is not working at the moment though.

### CODE REVIEW

**Comment by vaxerski** [comment]
Thanks. Completely forgot. Yeah, I know, it's in the issue tracker. Bug.

**Comment by vaxerski** [comment]
tag me when ready to merge (looks ready to me?)

**Comment by Aqa-Ib** [comment]
Alright, check if it is well written, or if you can explain it better than I did @vaxerski

**Comment by vaxerski** [comment]
I would not make a new page. I would just write a quick inline description in the variables page.

**Comment by vaxerski** [comment]
something like (probably in dispatchers is the best) `togglegroup | toggles a group. a group is a ... blah blah`

**Comment by Aqa-Ib** [comment]
Check it when you can please.

File: pages/Configuring/Dispatchers.md [hunk 1]
```diff
@@ -70,6 +70,22 @@ bind = MOD,KEY,exec,sleep 1 && hyprctl dispatch dpms off
...
 
+{{< hint type=info >}}
+
+## Grouped (tabbed) windows
+
+Hyprland allows you to make a group with the `togglegroup` bind dispatcher.
+
+A group is like i3wm’s “tabbed” container. It takes the space of one window, and you can change the window to the next one in the tabbed “group” with the `changegroupactive` bind dispatcher.
+
+When creating the group, a group will be made from the current active window. Groups can be tiled, floating, and can be used in dwindle and master layouts.
```
Comment (vaxerski) [inline pages/Configuring/Dispatchers.md:81]: layout note is redundant, they work regardless of the layout while this suggests they work _only_ on master and dwindel.

File: pages/Configuring/Dispatchers.md [hunk 2]
```diff
@@ -70,6 +70,22 @@ bind = MOD,KEY,exec,sleep 1 && hyprctl dispatch dpms off
...
+
+The new group’s border colors are configurable with the appropriate col. settings in the general config section.
+
+You can lock a group with the `lockgroups` bind dispatcher for an improved window management.
```
Comment (vaxerski) [inline pages/Configuring/Dispatchers.md:85]: for ~~an~~ improved window management

**Review by vaxerski — CHANGES_REQUESTED** [review]

**Comment by vaxerski** [comment]
also you forgot to add the group dispatchers to the dispatcher list. You yote them from dwindle and didnt readd back

**Comment by Aqa-Ib** [comment]
My bad. Fixed now, check it out please.

**Review by vaxerski — APPROVED** [review]
lgtm xd

**Comment by Aqa-Ib** [comment]
Thanks and sorry for the trouble. Next time I will try to do it better than this.

**Comment by vaxerski** [comment]
no worries

**Pull request merged**